### ISSUE TITLE
Planilha Lucro presumido

### ISSUE THREAD

**Issue by lcguida** [DyegoCosta/trabalhando-remoto#87]
Fala galera,

Faz mais ou menos um ano que eu estou trabalhando remoto pro exterior. Eu levei algum tempo no início para entender o sistema de tributação e conseguir calcular salário líquido e impostos à pagar.

Na ocasião eu fiz uma planilha pra calcular todas essas coisas, além de ter uma noção do custo mensal da empresa, qual valor do lucro e pró-labore à transferir, etc.

Depois de um tempo melhorando a planilha, acho que ela está num estado aceitável pra compartilhar: https://docs.google.com/spreadsheets/d/1q7yaJDS8m7JxZb2INpy8doCtAnt0uNBn8k2CozXJ4XY/edit?usp=sharing

Esse modelo que eu to compartilhando, calcula todos os impostos. Mas se você for isento de alguns (como PIS, CONFIS e ISS, que é o meu caso) basta zerar esses campos.

**Comment by yuriploc** [comment]
Belo trabalho, @lcguida , obrigado por compartilhar!

**Comment by brenopolanski** [comment]
Excelente trabalho @lcguida :clap:

**Comment by g13ydson** [comment]
Serve para esse caso? Vi que não é necessário alguns impostos como Dev para o exterior. https://medium.com/@hnordt/como-pagar-6-de-impostos-dentro-da-lei-a91c23868ec6

**Comment by lcguida** [comment]
Não. A planilha é para lucro presumido. O fator R, descrito no artigo, é para Simples Nacional.

**Comment by raulfragoso** [comment]
Passando aqui apenas para agradecer pela planilha. 8 meses depois e ainda a utilizo para o cálculo dos impostos mensalmente.

**Comment by lcguida** [comment]
Haha, valeu. Já faz quase 1 ano e meio que eu to usando. Me ajuda muito. 
Só atenção que eu atualizei ela com algumas correções. Achei alguns erros na versão anterior.

Mas ajuda bastante no planejamento e ver que os valores batem certinhos com as DARF's.

**Comment by raulfragoso** [comment]
Sim, todo mês o contador manda os DARF's e dá uma satisfação enorme ver até os centavos batendo 😀

**Comment by athos-ribeiro** [comment]
Sobre lucro presumido:

Eu vi em algumas fontes (e.g., https://receita.economia.gov.br/orientacao/tributaria/declaracoes-e-demonstrativos/ecf-escrituracao-contabil-fiscal/perguntas-e-respostas-pessoa-juridica-2018-arquivos/capitulo-xiii-irpj-lucro-presumido-2018.pdf/view), que quem tem receita oriunda do exterior não pode optar pelo lucro resumido, a menos que esteja prestando serviços de maneira direta no exterior.

Ainda assim eu vejo que algumas pessoas usam lucro presumido aqui para prestar serviços para fora.

Minha dúvida é sobre a interpretação do texto acima aí. O que vocês entendem por prestar serviço de maneira direta? Vocês (que se enquadram no exemplo acima) vivem no exterior?

Obrigado!

**Comment by lcguida** [comment]
Tirado esse texto no seu link:

> Não se  considera  direta  a  prestação  de  serviços  realizada  no exterior   por   intermédio   de   filiais,   sucursais,   agências, representações,   coligadas,   controladas   e   outras   unidades descentralizadas    da    pessoa    jurídica    que    lhes    sejam assemelhadas.

Acredito que o sentido de _direra_ tenha à ver com o fato de você trabalhar diretamente com a empresa e não através de um intermediário. Isso não significa que você tenha que você tenha que trabalhar "diretamente" no exterior.

**Comment by igorlvicente** [comment]
Por que você tem Lucro Presumido ao invés de Simples Nacional? É uma opção ou obrigação?

**Comment by lcguida** [comment]
É uma opção. No geral, se você vai trabalhar 100% pra fora, é fiscalmente mais vantajoso (à depender do salário também, mas com o dólar onde tá, geralmente é).

No Lucro presumido, fazendo exportação de serviço, você é isento de PIS e CONFINS e, dependendo do local, de ISS também, sobrando apena IRPJ e CSLL. Adicionar à isso a contribuição patronal de 20% em cima do seu pró-labore. Esses valores giram em torno de 8% ~ 9% no meu caso.

Já no simples você em uma alíquota calculada de acordo com o anexo da sua empresa. Nós nos encaixamos no Anexo V que tem uma alíquota um pouco acima de 15% ou 17%. Existe a possibilidade de baixar através do fator R, que é pagar pelo menos 28% do seu faturamento em pró-labore, passando do Anexo V ao Anexo III. Só que aí tem que considerar o aumento do seu pró-labore vai aumentar o seu imposto de pessoa física.

Então é calcular e ver qual fica melhor. Além disso tem que acompanhar as mudanças das leis. O que eu te disse é o que eu tenho conhecimento no momento, não sei se mudou algo desde que eu fiz essa escolha há 2 anos atrás.

**Comment by copercini** [comment]
@igorlvicente tem uma issue com essa discussão de simples x lucro presumido aqui: https://github.com/DyegoCosta/trabalhando-remoto/issues/54

a última vez que eu fiz as contas (em 2019, mas não deve ter mudado muito), para exportação de serviços, o simples vale mais a pena até R$21.000,00 de faturamento mensal, acima disso o lucro presumido começa valer mais a pena, mas tem mais (ainda mais) burocracia, ali no link tem os detalhes dos cálculos

**Comment by raulfragoso** [comment]
> Por que você tem Lucro Presumido ao invés de Simples Nacional? É uma opção ou obrigação?

Mera opção, mas você tem que pesar o que é melhor no seu caso. Como já mencionado acima, dependendo do seu faturamento, LP vai compensar mais que Simples. No meu caso, depois de botar na balança, optei por LP e, como tenho isenção do ISS no meu município (além do PIS/COFINS por conta da exportação de serviços), minha média de impostos fica em 8,7%

**Comment by marcelolima** [comment]
@lcguida muito obrigado pela planilha, comecei a usar há 2 meses e está me ajudando muito.

PS: acredito que só os deuses chegam nesse patamar, mas caso o lucro presumido fique acima de R$ 20.000,00 por mês, há um adicional no IRPJ de 10% sobre o que passar desse limite (não calculado na planilha no momento)

**Comment by lcguida** [comment]
Eu deixei de lado, pois para tirar isso, mesmo com o dólar à R$ 5,80 (como tá agora) passar desse valor seria um salário acima dos $130K. Tem senior que consegue tirar isso nos EUA (em empresa grande principalmente) mas não é mais raro.

Posso ver de adicionar isso depois.

**Comment by raulfragoso** [comment]
Pior que é o meu caso aqui, e eu nem me atentei a esse adicional. Vou falar com o contador.
Pelo que entendi, esses 10% são apurados trimestralmente sobre o excedente de 60k de lucro no trimestre (ou 20k por mês, pela média).

**Comment by lcguida** [comment]
Isso. É só se você passar na média do trimestre de R$ 20k/mês (R$ 60K/no total do semestre). Mas isso é o lucro presumido, ou seja, 32% do seu faturamento.

**Comment by raulfragoso** [comment]
Ahhhh tá, então o adicional é sobre o excedente do lucro presumido, e não do faturamento do trimestre. Então não é o meu caso (infelizmente hehehe)

**Comment by lcguida** [comment]
Isso. É em cima do Lucro Presumido, 32% do seu faturamento. Para pagar esse adicional você teria que tirar em média R$ 62.5K por mês (32% de R$ 62.5K = R$ 20k). Por isso que disse que não acho que é o caso da maioria.

**Comment by marcelolima** [comment]
@lcguida, podias me tirar uma dúvida? na parte de Receitas, vc coloca o valor que vai na Nota Fiscal (no dia que é emitida, junto com a invoice) ou o valor recebido em conta?  (visto que eles podem ser diferentes, caso o valor recebido em conta aconteça dias após a emissão da NFe)

**Comment by lcguida** [comment]
Eu coloco o valor recebido, pois é o que vai entrar na sua conta.

**Comment by RangelReale** [comment]
Eu li nesse site que, em lucro presumido, empresa de desenvolvimento de software tem que pagar 1% de RAT (risco de acidente de trabalho), é isso mesmo?
[https://dpobjetivo.com.br/tabelas/atribuicoes-cnae/details/1/1006](https://dpobjetivo.com.br/tabelas/atribuicoes-cnae/details/1/1006)

**Comment by rafaelnobrekz** [comment]
Estou me desenquadrando do Simples nacional esse mês. Vocês tiram prolabore de 1 salário? Ou nenhum ? Algum risco em não tirar ? Alguma vantagem (além da nossa pífia previdência)?

**Comment by nobre84** [comment]
Estou me desenquadrando do Simples nacional esse mês. Vocês tiram prolabore de 1 salário? Ou nenhum ? Algum risco em não tirar ? Alguma vantagem (além da nossa pífia previdência)?

**Comment by nobre84** [comment]
@lcguida Uma contribuição pra levar em conta o adicional de 10% de IR após 60k trimestrais (quem me dera, ainda chego lá!)
`=MAX((((B6+C6+D6)*0.32)-60000)*0.1,0)*-1`

**Comment by raulfragoso** [comment]
> Estou me desenquadrando do Simples nacional esse mês. Vocês tiram prolabore de 1 salário? Ou nenhum ? Algum risco em não tirar ? Alguma vantagem (além da nossa pífia previdência)?

Sim, pró-labore de 1 salário mínimo (atualmente em R$ 1.212,00). Acho que é obrigatório para sócio de empresa que não se enquadra no Simples, mas não tenho certeza. Eu não vejo vantagem alguma, e se não fosse compulsório, não teria.

**Comment by lcguida** [comment]
Um contador me disse uma vez que não é obrigatório, mas que não fazer pode dar problema na receita. Você teria que verificar isso com o seu contator.

> @lcguida Uma contribuição pra levar em conta o adicional de 10% de IR após 60k trimestrais (quem me dera, ainda chego lá!)
=MAX((((B6+C6+D6)*0.32)-60000)*0.1,0)*-1

Interessante. Eu não uso mais a planilha, sai do Brasil esse ano, então não estou mais usando a planilha. Mas posso colocar lá depois.
